msteams-platform/bots/what-are-bots.md

```diff
@@ -20,17 +20,17 @@ A bot is also referred to as a chatbot or conversational bot. It is an app that
 
 Conversational bots allow users to interact with your web service using text, interactive cards, and task modules.
 
-:::image type="content" source="../assets/images/invokebotwithtext.png" alt-text="Web service using text"lightbox="../assets/images/invokebotwithtext.png":::
+:::image type="content" source="../assets/images/invokebotwithtext.png" alt-text="The screenshot is an example that shows the conversational bot interacting with web service using text."lightbox="../assets/images/invokebotwithtext.png":::
```

Comment:
I believe the lack of space between end of alt-text parameter and beginning of lightbox parameter doesn't affect output. For now, that is. Update and add a space anyhow, so that tomorrow we don't run into Mardown syntax issues due to lack of a space.

Same comment applies to all other updates across all PRs.

Reply:
okay